packages/engine/tests/simulations/message_sending/views/analysis.json

```diff
@@ -0,0 +1,4 @@
+{
+  "outputs": {},
```

Comment:
Not important but I'm pretty sure this file can just be an empty JSON object.

Reply:
I will test this and report it back. If it's not needed, I'll replace this.